examples/tutorials/text.py

```diff
@@ -0,0 +1,139 @@
+"""
+Plotting text
+=============
+
+It is often useful to add annotations to a map plot. This is handled by
+:meth:`pygmt.Figure.text`.
+"""
+
+import os
+import pygmt
+
+###############################################################################
+# Basic map annotation
+# --------------------
+#
+# Text annotations can be added to a map using the :meth:`text` method of the
+# :class:`pygmt.Figure`.
+#
+# Here we create a simple map and add an annotation using the ``text``, ``x``,
+# and ``y`` arguments to specify the annotation text and position in the
+# projection frame. ``text`` accepts 'str' types, while ``x``, and ``y``
+# accepts either 'int' or 'float' types.
+
+fig = pygmt.Figure()
+with pygmt.config(MAP_FRAME_TYPE="plain"):
+    fig.basemap(region=[108, 120, -5, 8], projection="M20c", frame="a")
+fig.coast(land="black", water="skyblue")
+
+# Plotting text annotations using single elements
+fig.text(text="SOUTH CHINA SEA", x=112, y=6)
+
+# Plotting text annotations using lists of elements
+fig.text(text=["CELEBES SEA", "JAVA SEA"], x=[119, 112], y=[3.25, -4.6])
+
+fig.show()
+
+###############################################################################
+# Changing font style
+# -------------------
+# The size, family/weight, and colour of an annotation can be specified using
```

Comment:
Let's use US spellings.
```suggestion
# The size, family/weight, and color of an annotation can be specified using
```

Reply:
Personally I prefer the British spelling, but ok to go with this, if only because we use `color` as a long alias argument in many of our PyGMT modules.